config.json

```diff
@@ -0,0 +1,541 @@
+          "settings": {
+            "CLRTestPriorityToBuild": "default"
+          }
+        },
+        "sequential": {
+          "description": "Sets sequential build",
+          "settings": {
+            "maxcpucount": "default"
+          }
+        },
+        "skipRestore": {
+          "description": "Skips restoring packages during build.",
+          "settings": {
+            "RestoreDuringBuild": false
+          }
+        },
+        "BuildTarget": {
+          "description": "Runs the build target.",
+          "settings": {
+            "Build": "default"
+          }
+        }
+      },
+      "defaultValues": {
+        "toolName": "msbuild",
+        "settings": {
+          "MsBuildParameters": "default",
+          "MsBuildLog": "default",
+          "MsBuildWrn": "default",
+          "MsBuildErr": "default"
+        }
+      }
+    },
+    "sync": {
+      "alias": {
+        "p": {
+          "description": "Restores all NuGet packages for repository.",
+          "settings": {
+            "Project": "./build.proj",
+            "RestoreDuringBuild": true,
+            "RestoreNETCorePlatforms": "default"
+          }
+        },
+        "ab": {
+          "description": "Downloads the latests product packages from Azure. The values for '-AzureAccount' and '-AzureToken' are required",
+          "settings": {
+            "Project": "./src/syncAzure.proj"
+          }
+        },
+        "azureAccount": {
+          "description": "Account token to connect to Azure Blob storage.",
+          "settings": {
+            "CloudDropAccessToken": "default"
+          }
+        },
+        "azureToken": {
+          "description": "Account name to connect to Azure Blob storage.",
+          "settings": {
+            "CloudDropAccountName": "default"
+          }
+        },
+        "verbose": {
+          "description": "Passes /flp:v=diag to the msbuild command or the value passed by the user.",
+          "settings": {
+            "MsBuildFileLogging": "/flp:v=diag;LogFile=sync.log"
+          }
+        },
+        "buildMajor": {
+          "description": "To download a specific group of product packages, specify build number. The value for -BuildMinor required.",
+          "settings": {
+            "BuildNumberMajor": "default"
+          }
+        },
+        "buildMinor": {
+          "description": "To download a specific group of product packages, specify build number. The value for -BuildMajor required.",
+          "settings": {
+            "BuildNumberMinor": "default"
+          }
+        }
+      },
+      "defaultValues": {
+        "toolName": "msbuild",
+        "settings": {
+          "MsBuildParameters": "default",
+          "maxcpucount": "default",
+          "MsBuildFileLogging": "/flp:v=detailed;LogFile=sync.log"
+        }
+      }
+    },
+    "publish-packages": {
+      "alias": {
+        "azureAccount": {
+          "description": "Account token to connect to Azure Blob storage.",
```

Comment:
azureAccount is setting the property CloudDropAccessToken . It should set CloudDropAccountName, it makes more sense.
and then azureToken sets CloudDropAccessToken